sphinxcontrib/mermaid.py

```diff
@@ -234,24 +234,24 @@ def _render_mm_html_raw(
     self, node, code, options, prefix="mermaid", imgcls=None, alt=None
 ):
     if "align" in node and "zoom_id" in node:
-        tag_template = """<div align="{align}" id="{zoom_id}" class="mermaid align-{align}">
+        tag_template = """<pre align="{align}" id="{zoom_id}" class="mermaid align-{align}">
```

Comment:
the change of the element type, doesn't affect when zoom is enabled? check mermaid_d3_zoom

Reply:
Yes, it affects. Next commit with the fix